src/java/org/apache/cassandra/exceptions/RequestFailureReason.java

```diff
@@ -36,14 +39,17 @@ public enum RequestFailureReason
     TIMEOUT                  (2),
     INCOMPATIBLE_SCHEMA      (3),
     READ_SIZE                (4),
+    // below reason is only logged, but it does not have associated exception
     NODE_DOWN                (5),
     INDEX_NOT_AVAILABLE      (6),
+    // below reason does not have an associated exception
     READ_TOO_MANY_INDEXES    (7),
     NOT_CMS                  (8),
     INVALID_ROUTING          (9),
     COORDINATOR_BEHIND       (10),
-    ;
-
+    // below code is intentionally higher, coming as a back port from a fork. It was created that high to prevent
+    // any conflicts with Apache Cassandra code.
+    INDEX_BUILD_IN_PROGRESS  (503);
```

Comment:
I'm guessing this might not be the only one of these. Thinking of a general comment that takes that into account...maybe something like...

```
// The following codes have been ported from an external fork, where they were offset explicitly to avoid conflicts.
```